src/main/java/com/uniovi/entities/Incident.java

```diff
@@ -0,0 +1,106 @@
+package com.uniovi.entities;
+import com.uniovi.util.JasyptEncryptor;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+public class Incident {
+	
+	@Id @GeneratedValue
+	private Long id;
+	
+	private String username, password;
+	private String inciName, location;
+	private List<String> tags= new ArrayList<String>();
+	private List<String> moreInfo= new ArrayList<String>();
+
```

Comment:
"For some incidents, a list of fields can also be added with the form "property/value", where the property field refers to a property name and the value field refers to the value of that property."
I think we should also add a Map called properties in the incident class. I'm not sure if it would be a Map<String, String> or a Map<String, Object>, since we can store any kind of value in a property. I would use Map<String, Object> for now.